server/src/test/java/org/elasticsearch/rest/RestControllerTests.java

```diff
@@ -183,18 +183,18 @@ public void testRegisterAsDeprecatedHandler() {
         String path = "/_" + randomAlphaOfLengthBetween(1, 6);
         RestHandler handler = (request, channel, client) -> {};
         String deprecationMessage = randomAlphaOfLengthBetween(1, 10);
-        RestApiVersion previous = RestApiVersion.current().previous();
+        RestApiVersion current = RestApiVersion.current();
```

Comment:
I see how this become a problem.. Maybe deprecatedSinceVersion instead of current?

Reply:
+1, renamed in 44df5bf0433c647858e96b4cdae899251c602a45